src/core/stripAndCollapse.js

```diff
@@ -0,0 +1,12 @@
+define( function() {
+	"use strict";
+
+	// Strip and collapse whitespace according to HTML spec
+	// https://html.spec.whatwg.org/multipage/infrastructure.html#strip-and-collapse-whitespace
+	var rhtmlSpace = /[\x20\t\r\n\f]+/g,
```

Comment:
This is somehow different from `[ \t\r\n\v\f]` -> `\s` right? You guys already talked about this I believe?